include/networkit/centrality/DynApproxBetweenness.hpp

```diff
@@ -25,61 +25,68 @@ namespace NetworKit {
-      *
-      * @param	G			the graph
-      * @param	storePredecessors   keep track of the lists of predecessors?
-      * @param	epsilon		maximum additive error
-      * @param	delta		probability that the values are within the error guarantee
-      * @param	universalConstant	the universal constant to be used in
-      * computing the sample size. It is 1 by default. Some references suggest
-      * using 0.5, but there is no guarantee in this case.
+     * Implementation from the algorithm from "Approximating Betweenness Centrality in Fully-dynamic
+     * Networks" from Internet Mathematics vol. 5 by Elisabetta Bergamini and Henning Meyerhenke
+     * approximates the betweenness of all vertices so that the scores are within an additive error
+     * @a epsilon with probability at least (1- @a delta). The values are normalized by default.
+     *
+     * @param	G			the graph
+     * @param  storePredecessors   keep track of the lists of predecessors?
+     * @param	epsilon		maximum additive error
+     * @param	delta		probability that the values are within the error guarantee
+     * @param	universalConstant	the universal constant to be used in
+     * computing the sample size. It is 0.5 by default.
      */
     DynApproxBetweenness(const Graph &G, double epsilon = 0.01, double delta = 0.1,
-                         bool storePredecessors = true, double universalConstant = 1.0);
+                         bool storePredecessors = true, double universalConstant = 0.5);
 
     /**
      * Runs the static approximated betweenness centrality algorithm on the initial graph.
      */
     void run() override;
 
     /**
-    * Updates the betweenness centralities after an edge insertions on the graph.
-    * Notice: it works only with edge insertions and the graph has to be connected.
-    *
-    * @param e The edge insertions.
-    */
+     * Updates the betweenness centralities after an edge insertions on the graph.
+     *
+     * @param e The edge insertion or deletion.
+     */
     void update(GraphEvent e) override;
 
     /**
-    * Updates the betweenness centralities after a batch of edge insertions on the graph.
-    * Notice: it works only with edge insertions and the graph has to be connected.
-    *
-    * @param batch The batch of edge insertions.
-    */
-    void updateBatch(const std::vector<GraphEvent>& batch) override;
+     * Updates the betweenness centralities after a batch of edge insertions on the graph.
+     *
+     * @param batch The batch of edge insertions and/or deletions.
+     */
+    void updateBatch(const std::vector<GraphEvent> &batch) override;
 
     /**
-    * Get number of path samples used for last calculation
-    */
-    count getNumberOfSamples();
+     * Get number of path samples used for last calculation
+     */
+    count getNumberOfSamples() const noexcept;
+
+    std::vector<node> sortComponentsTopologically(Graph &sccDAG, StronglyConnectedComponents &scc);
+
+    count computeVDdirected();
```

Comment:
These two do not need to be public right?

Reply:
oh sorry, forgot to move them. Should be fixed now.